tests/test_validation_body.py

```diff
@@ -56,6 +57,20 @@ async def test_post_an_article_with_wrong_type_field_should_return_an_error_mess
     ]
 
 
+@pytest.mark.parametrize("request_body", [["foo"], "bar", 42])
+async def test_post_json_with_non_object_top_structure_should_return_an_error_message(
```

Comment:
Please could you add a test to verify that we can accept a JSON list in the body ? 
 
```
class ArticleModels(BaseModel):
    __root__: List[ArticleModel]

    def __iter__(self):
        return iter(self.__root__)
 
 class ArticleView(PydanticView):
     async def post(self, article: ArticleModel):
         return web.json_response(article.dict())
 
    async def put(self, articles: ArticleModels):
        return web.json_response(article.dict() for article in articles)
```

Thanks a lot!